internal/netsync/manager.go

```diff
@@ -1760,17 +1764,16 @@ func (m *SyncManager) RequestFromPeer(p *peerpkg.Peer, blocks, txs []*chainhash.
 }
 
 func (m *SyncManager) requestFromPeer(p *peerpkg.Peer, blocks, txs []*chainhash.Hash) error {
-	msgResp := wire.NewMsgGetData()
-
-	state, exists := m.peerStates[p]
-	if !exists {
+	peer, ok := m.peers[p]
```

Comment:
The warning in `lookupPeer` makes it adequate for receiving purposes only. Perhaps the warning could be updated to allow using it here as well for consistency.

Reply:
Good point.  I changed this out to just go ahead and log a stack trace in the even a lookup fails since that is probably a lot more useful anyway.